<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QWidget" name="categoriesWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="categoriesFilter"/>
          </item>
          <item>
           <widget class="QTreeWidget" name="categoriesTree">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>5</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="verticalScrollMode">
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>1</string>
             </property>
            </column>
            <item>
             <property name="text">
              <string>General</string>
             </property>
             <property name="statusTip">
              <string>generalPage</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/icons/img/config_general.png</normaloff>:/icons/img/config_general.png</iconset>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Font</string>
             </property>
             <property name="statusTip">
              <string>FontPage</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/icons/img/config_font.png</normaloff>:/icons/img/config_font.png</iconset>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Style</string>
             </property>
             <property name="toolTip">
              <string/>
             </property>
             <property name="statusTip">
              <string>stylePage</string>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/icons/img/config_style.png</normaloff>:/icons/img/config_style.png</iconset>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>5</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <widget class="QWidget" name="generalPage">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QGroupBox" name="dataBrowsingGroup">
             <property name="title">
              <string>Data browsing and editing</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="QLabel" name="rowsPerPageLabel">
                <property name="text">
                 <string>Number of data rows per page:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="rowsPerPageSpin">
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>99999</number>
                </property>
                <property name="cfg" stdset="0">
                 <string notr="true">General.NumberOfRowsPerPage</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="initColWidthLimitLabel">
                <property name="toolTip">
                 <string>&lt;p&gt;When the data is read into grid view columns width is automatically adjusted. This value limits the initial width for the adjustment, but user can still resize the column manually over this limit.&lt;/p&gt;</string>
                </property>
                <property name="text">
                 <string>Limit initial data column width to (in pixels):</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="initColWidthLimitSpin">
                <property name="toolTip">
                 <string>&lt;p&gt;When the data is read into grid view columns width is automatically adjusted. This value limits the initial width for the adjustment, but user can still resize the column manually over this limit.&lt;/p&gt;</string>
                </property>
                <property name="minimum">
                 <number>10</number>
                </property>
                <property name="maximum">
                 <number>99999</number>
                </property>
                <property name="value">
                 <number>600</number>
                </property>
                <property name="cfg" stdset="0">
                 <string notr="true">General.MaxInitialColumnWith</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>GroupBox</string>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="spinBox">
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>99999</number>
                </property>
                <property name="cfg" stdset="0">
                 <string>General.InsertRowPlacement</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="stylePage">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="activeStyleWidget" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="activeStyleLabel">
                <property name="text">
                 <string>Current style:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="activeStyleCombo">
                <property name="cfg" stdset="0">
                 <string notr="true">General.Style</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="activeStylePreviewGroup">
             <property name="title">
              <string>Preview</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QTabWidget" name="previewTabs">
                <property name="currentIndex">
                 <number>0</number>
                </property>
                <widget class="QWidget" name="previewTab1">
                 <attribute name="title">
                  <string>Enabled</string>
                 </attribute>
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <item>
                   <widget class="QWidget" name="previewWidget" native="true">
                    <layout class="QGridLayout" name="gridLayout_8">
                     <property name="leftMargin">
                      <number>0</number>
                     </property>
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <property name="rightMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                     <item row="2" column="0">
                      <widget class="QProgressBar" name="previewProgressBar">
                       <property name="value">
                        <number>24</number>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="6">
                      <widget class="QCheckBox" name="previewCheckBox">
                       <property name="text">
                        <string notr="true">CheckBox</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1" rowspan="4">
                      <widget class="QSlider" name="previewVerticalSlider">
                       <property name="value">
                        <number>50</number>
                       </property>
                       <property name="orientation">
                        <enum>Qt::Vertical</enum>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="2" rowspan="4">
                      <widget class="QScrollBar" name="previewVerticalScrollBar">
                       <property name="orientation">
                        <enum>Qt::Vertical</enum>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="3" colspan="2">
                      <widget class="QPushButton" name="previewPushButton">
                       <property name="text">
                        <string notr="true">PushButton</string>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="3" colspan="3">
                      <widget class="QSpinBox" name="previewSpinBox"/>
                     </item>
                     <item row="2" column="3" colspan="5">
                      <widget class="QProgressBar" name="progressBar">
                       <property name="maximum">
                        <number>0</number>
                       </property>
                       <property name="value">
                        <number>0</number>
                       </property>
                       <property name="alignment">
                        <set>Qt::AlignCenter</set>
                       </property>
                      </widget>
                     </item>
                     <item row="3" column="0">
                      <widget class="QTreeWidget" name="previewTreeWidget">
                       <column>
                        <property name="text">
                         <string notr="true">Column</string>
                        </property>
                       </column>
                       <item>
                        <property name="text">
                         <string notr="true">123</string>
                        </property>
                        <item>
                         <property name="text">
                          <string notr="true">11111</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string notr="true">22222</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string notr="true">33333</string>
                         </property>
                        </item>
                       </item>
                       <item>
                        <property name="text">
                         <string notr="true">456</string>
                        </property>
                        <item>
                         <property name="text">
                          <string notr="true">44444</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string notr="true">55555</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string notr="true">66666</string>
                         </property>
                        </item>
                       </item>
                      </widget>
                     </item>
                     <item row="0" column="5">
                      <widget class="QToolButton" name="previewToolButton">
                       <property name="text">
                        <string notr="true">...</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="0">
                      <widget class="QLineEdit" name="previewLineEdit"/>
                     </item>
                     <item row="0" column="6">
                      <widget class="QRadioButton" name="previewRadioButton">
                       <property name="text">
                        <string notr="true">RadioButton</string>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="0">
                      <widget class="QComboBox" name="previewComboBox">
                       <property name="currentText">
                        <string notr="true">ABC</string>
                       </property>
                       <item>
                        <property name="text">
                         <string notr="true">ABC</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string notr="true">XYZ</string>
                        </property>
                       </item>
                      </widget>
                     </item>
                     <item row="3" column="3" colspan="5">
                      <widget class="QTextEdit" name="previewTextEdit">
                       <property name="html">
                        <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt;&quot;&gt;Abcdefgh&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="previewTab2">
                 <attribute name="title">
                  <string>Disabled</string>
                 </attribute>
                 <layout class="QGridLayout" name="gridLayout_7"/>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="FontPage">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>59</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>447</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
